package.json

```diff
@@ -14,6 +14,7 @@
     "@reduxjs/toolkit": "1.3.4",
     "@sentry/browser": "5.15.4",
     "@walletconnect/web3-provider": "1.4.1",
+    "amplitude-js": "^8.10.0",
```

Comment:
Let's pin this dependency

Trying to pin all deps moving forward to reduce probability of supply chain attacks